src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int bottomBase;
+    private int topBase;
+    private int hight;
+
+    public IsoscelesTrapezoid(int bottomBase, int topBase, int hight, Color color) {
+        this.bottomBase = bottomBase;
+        this.topBase = topBase;
+        this.hight = hight;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return ((bottomBase + topBase) / 2) * hight;

```

Comment:
Ensure that the calculation in `getArea()` is correct according to the formula for the area of an isosceles trapezoid: `((bottomBase + topBase) / 2) * height`.